packages/replayio/src/utils/async/timeoutAfter.ts

```diff
@@ -0,0 +1,15 @@
+type TimeoutResult = { timedOutAfter: number };
+
+export async function timeoutAfter(duration: number): Promise<TimeoutResult> {
```

Comment:
I liked the previous `wait` (the name worked especially great in situations like in the retrying functions). `timeoutAfter` is cool but my intuition was that it's supposed to throw. I think throwing would probably overcomplicate what happens in `initialize`:
https://github.com/replayio/replay-cli/blob/cli-internet-connectivity/packages/replayio/src/utils/initialization/initialize.ts#L20-L26

So I don't know if it's worth it. Or maybe we could use some `throwOnTimeout = true` option? It's totally subjective so pick whatever you feel is best here.

Reply:
Yeah I was torn about the name change also but leaned slightly toward this one. I also considered a `throwOnTimeout` but worried the forking behavior may have been confusing. I don't mind adding it but I don't think we'll use it anywhere.